uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/utils/ProjectFactory.java

```diff
@@ -162,13 +162,13 @@ public static void importExistingProject(String path, String name, boolean maven
 	 * 
 	 * @return all available templates
 	 */
-	public static List<String> getAllAvailableTemplates() {
+	public static List<FuseProjectDefinition> getAllAvailableTemplates() {
 		List<String> versions = getAllAvailableCamelVersions();
-		List<String> templates = new ArrayList<String>();
+		List<FuseProjectDefinition> templates = new ArrayList<FuseProjectDefinition>();
```

Comment:
you can use the diamond operator to write in a shorter way: List\<FuseProjectDefinition> templates = new ArrayList\<>();

Reply:
thank you :+1: